docs/core/testing.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Testing
+order: 6
+---
+
+<a name="testing"></a>
+
+# Testing
+
+When testing your components, you're likely going to want to check arguments and force different states from the Urql hooks, this is the place to get started.
+
+## Communicating with hooks
```

Comment:
I think this section we can remove. That’d mean that we can explicitly leave a note in the above introduction saying: “This page will show some examples with `urql`’s React hook. But the same testing approach applies. Mocking the client is a solid way to test the behaviour of the rest of your codebase.”

Then this page is pretty future proof